target/consensource
# Update Sawtooth to 1.2

- Update `sawtooth-sdk` package in `Cargo.toml`s and `Cargo.lock`s
- Change `Dockerfile`s to use new rust image and do expose builds
- Add build steps to client
- Update images in `docker-compose.yaml`
- Add required consensus service

## Proposed change/fix

Necessary updates to run on Sawtooth 1.2 instead of 1.0

## Types of changes

What types of changes is this pull request introducing to ConsenSource? _Put an `x` in the boxes that apply_


## How to run/test

Build the client locally if you haven't already:
``` 
cd client
npm i
npm run build
```

Then: 
`docker-compose up --build`, then verify transactions are able to be processed from requests sent from the client.
